packages/backend/src/apps/formsg/triggers/new-submission/get-data-out-metadata.ts

```diff
@@ -73,6 +73,7 @@ function buildAnswerArrayForCheckbox(
         ? `Response ${fieldData.order}, Selected Option ${i + 1}`
         : null,
       order: fieldData.order ? (fieldData.order as number) : null,
+      isHidden: true,
```

Comment:
what will happen to the old pipes which are using answer array, and what is the plan to handle user questions or confusion regarding these changes

Reply:
ok got it, this may be a little more complicated than i thought, will check in with yall